src/discrete-layer/components/catalog-tree/catalog-tree.tsx

```diff
@@ -119,7 +120,7 @@ export const CatalogTreeComponent: React.FC<CatalogTreeComponentProps> = observe
 
   const entityPermittedActions = useMemo(() => {
     const entityActions: Record<string, unknown> = {};
-    ['LayerRasterRecord', 'Layer3DRecord', 'BestRecord', 'LayerDemRecord',].forEach( entityName => {
+    ['LayerRasterRecord', 'Layer3DRecord', 'BestRecord', 'LayerDemRecord','VectorBestRecord'].forEach( entityName => {
```

Comment:
open issue in Jira in order to refactor
might be place d in _entity-types-keys.ts_

Reply:
Opened an Isuue on JIra 'MAPCO-1586'